model/dialogue/hmnet/ExampleRawData/meeting_summarization/AMI_proprec/test_ami.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Why do we need this file? Is it possible use one of the datasets instead, now that the datasets are merged?

Reply:
We need this file because HMNet requires a specific file to restore a brief introduction to the dataset. However, I will delete this file in the next version.